env.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import print_function
```

Comment:
The rest of the clcache script code now requires Python 3.3 or newer. I think imposing the same requirement here would be consistent, and it would simplify things abit (e.g. this import is not needed).